src/apps/logging/logging.lua

```diff
@@ -10,11 +10,11 @@ return {
         METHOD_NOT_FOUND = -32601,
         INVALID_PARAMS = -32602,
         INTERNAL_ERROR = -32603,
-        INVALID_CLIENT_SIGNATURE = -32605,
-        INVALID_CALLER_ID = -32606,
-        
-        INSUFFICIENT_RIGHTS = -32006,
-        DENIED = -32007
+
+        INSUFFICIENT_RIGHTS = -32004,
+
```

Comment:
What happened to 5 and 6?

Reply:
They were renumbered, and I only kept the values which were previously here. For simplicity, I've now repeated the full list of errors from C++

Reply:
Ok